src/actions/l10n.js

```diff
@@ -26,10 +27,16 @@ export function requestL10n(): Action {
 /**
  * Receive translations for a locale.
  */
-export function receiveL10n(localization: Localization): Action {
+export function receiveL10n(
+  localization: Localization,
+  primaryLocale: string,
+  direction: 'ltr' | 'rtl'
```

Comment:
yeah, I thought of this but wasn't sure.

Mainly having it here explicitely without a named type is useful because it's very clear, when looking at the function, what the possible values are.

(or maybe I should switch to an editor that does that for me?)

Reply:
> (or maybe I should switch to an editor that does that for me?)

😄 

Yeah, I think it's also fine to leave it that way, but I usually like to keep them in a single type. Feel free to follow my suggestion or not.